projectile.el

```diff
@@ -3300,6 +3300,13 @@ entirely."
   :risky t
   :package-version '(projectile "0.12.0"))
 
+(defun projectile-find-file-hook-function ()
+  "Called by `find-file-hook' when `projectile-mode' is on."
+  (unless (file-remote-p default-directory)
```

Comment:
As mentioned in the ticket - the lighter causes the remote delay. It has nothing to do with those hooks.